README.md

```diff
@@ -102,6 +102,17 @@ This command will help migrate the database and populate the database!
 php artisan migrate --seed
 ```
 
+If an error of permission denied occurs in MySQL, try creating a non-root user with a password and update the .env file as required. Also make sure that the database name provided in the .env file, actually exists in the MySQL databases. You can find detailed solution [here](https://stackoverflow.com/questions/46680785/laravel-artisan-access-denied-for-rootlocalhost-on-console-only/47116614#47116614)
+
+After configuration, the .env file should look somewhat similar like this
+```DB_CONNECTION=mysql
+DB_HOST=localhost
+DB_PORT=3306
```

Comment:
This should go to the troubleshoot.md and also it is mentioned to replace all necessary values after doing the 
`cp .env-example .env`